Cargo.toml

```diff
@@ -17,7 +17,8 @@ members = [
 
 [patch.crates-io.maitake]
 git = "https://github.com/hawkw/mycelium.git"
-rev = "5149ee6146a9e0c1af56da79bb55578abf6dfddc"
+# rev = "5149ee6146a9e0c1af56da79bb55578abf6dfddc"
+branch = "eliza/next-deadline"
```

Comment:
Reminder to rev after the mycelium PR lands

Reply:
oh, yeah, this can be pointed back to main